astroplan/plots/time_dependent.py

```diff
@@ -145,40 +148,47 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, Sequence):
+        targets = [targets]
 
-    # Some checks & info for labels.
-    if not hasattr(target, 'name'):
-        target_name = ''
-    else:
-        target_name = target.name
-    style_kwargs.setdefault('label', target_name)
+    for target in targets:
+        # Calculate airmass
+        airmass = observer.altaz(time, target).secz
+        # Mask out nonsense airmasses
+        masked_airmass = np.ma.array(airmass, mask=airmass < 1)
 
-    # Plot data.
-    ax.plot_date(time.plot_date, masked_airmass, **style_kwargs)
+        # Some checks & info for labels.
+        if not hasattr(target, 'name'):
```

Comment:
I know that this is the original style but wouldn't a `try .. except ..` make more sense?

```
try:
    target_name = target.name
except AttributeError:
    target_name = ''
```